### DISCUSSION TITLE
let's Encrypt and MISP

### DISCUSSION THREAD

**Discussion by Linow974** [MISP/MISP#8131] [General]
Hello,

I recently installed MISP on an LXC container.

Everything works fine, I access through a web interface to MISP via https://192.168.x.x ...

The problem is that the SSL certificate configured by default is not certified. There is therefore a warning page that is displayed, before accessing MISP. This can be annoying for any use towards the public.

So I tried setting up a signed certificate with Let's Encrypt. However, the MISP URL name is an IP address, and Let's Encrypt apparently doesn't support IP address domains.

So I researched how to change MISP URL, MISP domain, I found this:

https://github.com/MISP/misp-book/issues/196

But it doesn't change access to MISP, I think I need to change some settings in apache, but I don't know how.

Any ideas ?

**Comment by adulau** [comment]
For let's encrypt and proper X.509 certificate, you need to assign a host and domain for your MISP instance.

When it's done, you just need to change the baseurl of your installation in the user-interface matching the certificate DN.
